pkg/controller/networkpolicy/validate.go

```diff
@@ -637,20 +637,23 @@ func (t *tierValidator) deleteValidate(oldObj interface{}, userInfo authenticati
 }
 
 // validateAntreaGroupSelectors ensures that an IPBlock is not set along with namespaceSelector and/or a
-// podSelector.
+// podSelector. Similarly, ExternalEntitySelector cannot be set with PodSelector.
 func validateAntreaGroupSelectors(s corev1a2.GroupSpec) (string, bool) {
-	podSelector, serviceRef, ipBlock := 0, 0, 0
-	if s.NamespaceSelector != nil || s.PodSelector != nil {
-		podSelector = 1
+	selector, serviceRef, ipBlock := 0, 0, 0
+	if s.NamespaceSelector != nil || s.ExternalEntitySelector != nil || s.PodSelector != nil {
+		selector = 1
+	}
+	if s.PodSelector != nil && s.ExternalEntitySelector != nil {
```

Comment:
nit: make it first cond and return early?

Reply:
could do.. but probably don't want to wait for another 5hrs of CI :) i will follow up after the release

Reply:
Sounds good.